test/value_from.cpp

```diff
@@ -173,7 +181,10 @@ class value_from_test
     {
         // value_from supports every value constructor
 
-        testValueCtor<value const&>();
+        testValueCtor<value>();
+
+        char const* s = "5";
+        testValueCtor(s);
```

Comment:
It looks like these tests are incomplete? `testValueCtor` is only called from 1 or 2 places. I'm not sure what is supposed to be happening here.